e2e_samples/fabric_dataops_sample/docs/adr_naming_convention.md

```diff
@@ -0,0 +1,79 @@
+# Naming convention for Fabric E2E sample
+
+This document describes the naming convention for the Fabric E2E sample. It includes the naming convention for the following:
+
+- Git Folders
+- Scripts (python, shell, powershell etc.)
+- Configuration files (.yml, .json, .cfg etc.)
+- Azure/Fabric resources
+
+## Status
+
+Proposed
+
+## Context
+
+As the Fabric E2E sample is a complex solution with multiple components, it is important to have a consistent naming convention for the various components. This will help in better organization and management of the solution. It would also help multiple developers to work on the solution without any confusion.
+
+## Proposal details
+
+### GitHub files naming convention
+
+| Item Type | Convention | Additional Details | Example |
+| --- | --- | --- | --- |
+| Folders | snake_case | Use lowercase and "_" | `fabric_dataops_sample`, `dataops_pipeline` |
+| Markdown files | snake-case | Use lowercase and "_" | "README.md", `adr_naming_convention.md` |
```

Comment:
markdown files. (may be you should say upper or lower).. as standard .md files will all be in upper case (e.g README.md, CONTRIBUTING.md )

Reply:
Changed